.travis.yml

```diff
@@ -1,9 +1,6 @@
-
 language: go
-go: 
- - tip
-
+go:
+ - 1.6.3
```

Comment:
We did, but see:
- https://github.com/feedhenry/fh-system-dump-tool/pull/19#issuecomment-239110882
- https://travis-ci.org/feedhenry/fh-system-dump-tool/builds/151449259

Making Travis build Go tip to run our tests wastes our time.

We use 1.6.3 to build releases, that should be the version we test against IMHO.
I initially left both tip and 1.6.3 in there, but I think I prefer the fastest feedback of just 1.6.3.

Do you still see strong value in running all PRs against Go tip?